test/vttest_sample_test.py

```diff
@@ -145,6 +141,43 @@ def test_standalone(self):
     cursor.close()
     conn.close()
 
+    # Test we can connect to vtcombo for vtctl actions
+    out, _ = utils.run(
+        environment.binary_args('vtctlclient') +
+        ['-vtctl_client_protocol', protocol,
+         '-server', vtagte_addr,
+         '-stderrthreshold', 'INFO',
+         'ListAllTablets', 'test',
+        ], trap_output=True)
+    num_master = 0
+    num_replica = 0
+    num_rdonly = 0
+    num_dash_80 = 0
+    num_80_dash = 0
+    for line in out.splitlines():
```

Comment:
we do but it's in the vtctld client code, which this one is not. making it generic is too big of a hammer I think.